x-pack/plugins/siem/public/components/stat_items/index.tsx

```diff
@@ -49,28 +36,10 @@ export interface StatItem {
   description?: string;
   value: number | undefined | null;
   color?: string;
-  icon?: 'storage' | 'cross' | 'check' | 'visMapCoordinate';
+  icon?: 'storage' | 'cross' | 'check' | 'visMapCoordinate' | 'globe';
```

Comment:
Would this just be better as the exported Icon Type from EUI?:

```
import { IconType } from '@elastic/eui';
```

```ts
icon?: IconType
```

Then you don't have to retype it for each new icon you want to add here.

Reply:
Cool, that makes the code much easier to maintain! Thanks!